docs/adr/0009-modelling-history.md

```diff
@@ -1,6 +1,6 @@
 # 9. Modelling history
```

Comment:
I don't think we should be changing the ADR for this, it represents a document at a point in time rather than a living document.

Reply:
I did wonder - undone.